packages/frontend/src/components/common/Table/Table.styles.ts

```diff
@@ -106,8 +107,15 @@ export const TableFooter = styled.div`
     margin-top: 10px;
 `;
 
+const FontSyles = `
+    font-size: 13px;
+    font-family: Inter, sans-serif;
+`;
+
 const CellStyles = css<{ $isNaN: boolean }>`
     text-align: ${({ $isNaN }) => ($isNaN ? 'left' : 'right')} !important;
+    padding: 8.5px !important;
```

Comment:
do we have that in mantine table?

Reply:
No, this is the conversations from above about whether to use `rem`. In the pivot table this is `0.5rem`. This pixel version is probably pretty close.